src/renderer/pages/project/chat/graph.ts

```
@@ -56,16 +77,33 @@
               isResult: true,
               model: "gpt-4o",
             },
-            console: { after: true },
+            console: { before: true },
             inputs: {
               system: ":systemPrompt",
-              prompt: ":prompt",
+              prompt: [
+                "If there were errors in the previous generation, fix them!! Perfect.",
+                ":prompt"
+              ],
```

ESLint

[error] Replace `⏎················"If·there·were·errors·in·the·previous·generation,·fix·them!!·Perfect.",⏎················":prompt"⏎··············` with `"If·there·were·errors·in·the·previous·generation,·fix·them!!·Perfect.",·":prompt"`

(prettier/prettier)